### DISCUSSION TITLE
Welcome to react-pdf Discussions!

### DISCUSSION THREAD

**Discussion by diegomura** [diegomura/react-pdf#1082] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by roylee0704** [comment]
Hey Diegomura!

I have a question regarding react-pdf, does it renders PDF from file uri?

I've lookup-ed couple of places before deciding to post it here:
- GitHub issues
- Official landing page/Doc Components
- Repo examples

However the closest I could find is it only supports PDF rendering from the existing library's primitive `<Document/>` block.

**Reply by diegomura** [reply]
Hi @roylee0704 . You mean if it can be used to just visualize an existing document? No, that's not possible. This library is just for creating new documents
